### PR TITLE
PB-202 fix status updates

### PR DESCRIPTION
**Pull request by zanchin**

### CODE REVIEW

File: lib/BitbucketHandler.js [hunk 1]
```diff
@@ -460,11 +461,14 @@ BitbucketHandler.prototype.authLookupController = function(req, res, next) {
       },
     };
 
+    var self = this;
     var bitbucket = this.getBitbucketApi(project);
     bitbucket.refreshToken(function(err, auth) {
       if (err) {
         return next(err);
       }
+      self.log.info({auth: auth}, 'refreshed auth token from BB');
```
Comment (tortillaj) [inline lib/BitbucketHandler.js]: I would personally rather not store the token in the logs. I know with BB they are short-lived, but they could still be stolen ... if we want to have something traceable in the logs, maybe we could store the user ID? or something similar ...

File: lib/bitbucket/index.js [hunk 2]
```diff
@@ -212,6 +212,8 @@ Bitbucket.prototype.refreshToken = function(done) {
       return done(error);
     }
 
+    self.log.info({auth: body.access_token}, 'refreshed auth token from BB');
```
Comment (tortillaj) [inline lib/bitbucket/index.js]: same as the comment above re: tokens in logs

File: lib/BitbucketHandler.js (hunk 1 above)
Comment (zanchin) [inline lib/BitbucketHandler.js]: Agreed, will remove

File: lib/bitbucket/index.js (hunk 2 above)
Comment (zanchin) [inline lib/bitbucket/index.js]: Agreed, will remove

**Comment by zanchin** [comment]
@tortillaj removed those log lines

**Comment by tortillaj** [comment]
@ilya ok, LGTM

**Pull request merged**